modules/gdscript/gdscript.h

```diff
@@ -491,6 +491,8 @@ class GDScriptLanguage : public ScriptLanguage {
 	virtual void auto_indent_code(String &p_code, int p_from_line, int p_to_line) const override;
 	virtual void add_global_constant(const StringName &p_variable, const Variant &p_value) override;
 	virtual void add_named_global_constant(const StringName &p_name, const Variant &p_value) override;
+	virtual bool has_any_global_constant(const StringName &p_name) { return named_globals.has(p_name) || globals.has(p_name); }
+	virtual Variant get_any_global_constant(const StringName &p_name);
```

Comment:
these should probably not be virtual and get moved up to ~line 457 with the `get_global_map()` functions, because they're not a part of the editor functions

Reply:
Oh, good call, thank you! I wasn't even thinking when I put these there, I just looked around similar functions and didn't notice there were sections!